### PR TITLE
updates release workflow and gradle plugin

### PR DESCRIPTION
**Pull request by JBAhire**
## Description
addresses https://github.com/hypertrace/hypertrace/issues/19 & https://github.com/hypertrace/hypertrace/issues/22 

Please read discussion here: https://github.com/hypertrace/attribute-service/pull/30

### Testing
NA

### Checklist:
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -55,20 +55,21 @@ jobs:
...
+      - docker_login
+      - run: ./gradlew dockerPushImages
+  release-publish:
+    executor: gradle_docker
+    steps:
+      - setup_build_environment
+      - setup_remote_docker: *latest_remote_docker
       - docker_login
-      - run: ./gradlew :tag -Prelease
-      - run: cp -v /tmp/workspace/occollector_linux ./cmd/occollector/
```
Comment (aaron-steinfeld) [inline .circleci/config.yml:68]: I'm not spying this copy command in the new version, is it still needed?

Comment (JBAhire) [reply .circleci/config.yml:68]: ohh. I guess we do need it. it has dockerfile so yeah.

**Review by aaron-steinfeld — COMMENTED** [review]
LGTM once snyk resolved

**Comment by JBAhire** [comment]
@aaron-steinfeld , this synk issue looks like similar to what we observed in entity-service yesterday. can you please check once?

**Comment by aaron-steinfeld** [comment]
> @aaron-steinfeld , this snyk issue looks like similar to what we observed in entity-service yesterday. can you please check once?

It is a snyk-flagged vulnerability, but a completely different one. I'm not familiar with this code at all so I'll let @davexroth or @mohit-a21 take care of resolving these issues.

**Review by tim-mwangi — APPROVED** [review]

**Comment by JBAhire** [comment]
the snyk vulnerability is same as earlier one so @mohit-a21 said to ignore it for now.

**Pull request merged**